rootfs-builder/README.md

```diff
@@ -21,10 +25,23 @@ The `rootfs.sh` script builds a rootfs based on a particular Linux\*
 distribution. The script supports multiple distributions and can be extended
 to add further ones.
 
-To list the supported distributions, run:
+### Extra features
+#### Supported distributions list
+Supported distributions can be listed with:
+```
+$ ./rootfs.sh -l
+```
+
+#### Generate Kata specific files
+`rootfs.sh` can be used to populate a directory with only Kata specific files and
+components, without creating a full usable rootfs.
+This feature is used to create a rootfs based on a distribution not officially
+supported by osbuilder, and when building an image using the dracut build method.
 
+To achieve this, simply invoke `rootfs.sh` without specifying a target rootfs, e.g.:
 ```
-$ ./rootfs.sh -h
+$ mkdir kata-overlay
+$ ./rootfs.sh -r `pwd`/kata-overlay
```

Comment:
Nit: I'd try to avoid backticks in commands as they can be hard to see. Maybe use the variable instead:

```sh
$ ./rootfs.sh -r "$PWD/kata-overlay"
```